markdown/about.md

```diff
@@ -4,21 +4,75 @@ The sanger-tol project is the informatics branch of the Tree of Life programme a
 Here you can read how we organise ourselves, how we are funded and how the sanger-tol project was started.
 
 We welcome external contributions and collaboration on our projects.
-If you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
+If you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=muffato%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
 
 Please note that all community members are expected to adhere to our [code of conduct](/code_of_conduct), which is adopted from [nf-core](https://nf-co.re).
 
 ## Informatics Infrastructure team {#it}
 
-The [Informatics Infrastructure team](/it) provides support for the production of reference genome assemblies and large-scale genome analyses in the Tree of Life programme, and helps with the management and use of informatics resources and digital solutions. Current members are listed below:
+### Introduction {#it-intro}
 
-- [Matthieu Muffato](/it#matthieu-muffato)
-- [Guoying Qi](/it#guoying-qi)
-- [Priyanka Surana](/it#priyanka-surana)
-- [Cibin Sadasivan Baby](/it#cibin-sadasivan-baby)
-- [Cibele Sotero-Caio](/it#cibele-sotero-caio)
-- [Paul Davis](/it#paul-davis)
-- [Beth Yates](/it#beth-yates)
+The Informatics Infrastructure team provides support for the production of reference genome assemblies and large-scale genome analyses in the Tree of Life programme, and helps with the management and use of informatics resources and digital solutions.
+
+The team is organised in three poles.
+
+📂 **Data management**: Our data curators and managers maintain the integrity, consistency, and quality, or multiple databases used in production, including Genomes on a Tree (GoaT), Sample Tracking System (STS), Collaborative Open Plant Omics (COPO), and BioSamples.
```

Comment:
Can we add the links for each of the website? actually I don't know what COPO is and I need to search.

Reply:
Done